storage/objects/objects_test.go

```diff
@@ -257,7 +240,7 @@ func TestV4SignedURL(t *testing.T) {
 		t.Errorf("httpClient.Do: %v", err)
 	}
 	getBuf := new(bytes.Buffer)
```

Comment:
Maybe it'll be better to keep one style of defining buffer?
`var buf bytes.Buffer`